File: src/main/Scala/com/bjsxt/scalaspark/core/actions/Action_take.scala
package com.bjsxt.scalaspark.core.actions

import org.apache.spark.{SparkConf, SparkContext}

/**
  * 取出RDD中的前N个元素
  */
object Action_take {
  def main(args: Array[String]): Unit = {
    val conf = new SparkConf().setAppName("take").setMaster("local")
    val sc = new SparkContext(conf)
    val lines = sc.textFile("./data/words")
    val array = lines.take(3)
    array.foreach(println)
}
}

File: src/main/java/com/bjsxt/javaspark/core/actions/Action_take.java
package com.bjsxt.javaspark.core.actions;

import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaRDD;
import org.apache.spark.api.java.JavaSparkContext;

import java.util.List;

public class Action_take {
    public static void main(String[] args) {
        SparkConf conf = new SparkConf().setAppName("take").setMaster("local");
        JavaSparkContext sc = new JavaSparkContext(conf);
        JavaRDD<String> lines = sc.textFile("./data/words");
        List<String> resultList = lines.take(3);

        for(String one:resultList){
            System.out.println(one);
        }


    }
}
